spec/generators/rspec/controller/controller_generator_spec.rb

```diff
@@ -19,7 +19,7 @@
       describe 'the spec' do
         it { is_expected.to exist }
         it { is_expected.to contain(/require 'rails_helper'/) }
-        it { is_expected.to contain(/^RSpec.describe PostsController, :type => :controller/) }
+        it { is_expected.to contain(Regexp.new("^RSpec.describe PostsController, #{type_metatag(:controller)}")) }
```

Comment:
We use either `/regex/` or `%r{regex}` could you use one of those instead of `Regexp.new` please. :heart: